### DISCUSSION TITLE
Installation error: tflite_runtime-2.6.0-cp39-none-linux_aarch64.whl is not a supported wheel

### DISCUSSION THREAD

**Discussion by juddpatterson** [mcguirepr89/BirdNET-Pi#1095] [Q&A] [answered]
Brand new Raspberry Pi 4 model B and BirdNET Pi user here. I just installed RaspiOS-Lite (64-bit) and then ran the curl command from the installation guide. Things moved right along until I got:

```
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
ERROR: tflite_runtime-2.6.0-cp39-none-linux_aarch64.whl is not a supported wheel on this platform.
The installation exited unsuccessfully.
```
I checked for solutions and found https://github.com/mcguirepr89/BirdNET-Pi/discussions/847 and https://github.com/mcguirepr89/BirdNET-Pi/discussions/775 which mentioned mismatches in 32/64 bit and old versions of python. I don't believe any of that is what I'm experiencing. uname is reporting aarch64, my python version is 3.11.2. A few more details:

```
Python 3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform
>>> platform.architecture()
('64bit', 'ELF')
```

I'm not quite sure what to try next. Any thoughts?

**Comment by pddpauw** [comment, marked as answer]
I've experienced the same problem - see the bottom of #847 : the slightly older Bullseye 11.7 version did the trick to me (you can select the version in  the Raspberry Pi Imager).

**Comment by juddpatterson** [comment]
Thanks @pddpauw! That did the trick. Sorry I didn't pull that out of the other Q&A. I wasn't quite sure what Bullseye meant, but now I see that it is an older version of Raspberry Pi and you can choose it through the Imager.

**Reply by pddpauw** [reply]
no probs, welcome to the wonderful world of the Raspberry Pi's.  Enjoy the bird monitoring!

**Comment by Nakul2124** [comment]
The same problem came to me as i think it my python version but i cannot degrade my raspberry pi python version for some reason
